Source/WebCore/page/InteractionRegion.h

```diff
@@ -62,6 +62,9 @@ struct InteractionRegion {
     OptionSet<CornerMask> maskedCorners { };
     ContentHint contentHint { ContentHint::Default };
     std::optional<Path> clipPath { std::nullopt };
+#if ENABLE(INTERACTION_REGION_TEXT_CONTENT)
+    String text { };
```

Comment:
I think you can drop the initializer?

Reply:
Nope, because there are clients who partially initialize this struct, and that would make them have to specify all the earlier fields (I had that first!)

Reply:
(that's why all the fields above are similar)

Reply:
Note to self - actually read how the struct is used before commenting 🤣